views/index.ejs

```diff
@@ -8,8 +8,18 @@
     <h1>Shopping for Cookies</h1>
 
 
-    <strong>Items in cart: <!-- Print the number of items in the cart here --></strong>
+    <strong>Items in cart: <%= cart.length %></strong>
     <ul>
+      <% cart.forEach(function(product){ %>
+      <form action="'/cart'" method="post">
+        <li><%= product.name %></li>
+        <input type="hidden" name="_method" value="delete">
+        <input type="hidden" name="id" value="<%= product.id %>">
+        <input type="submit" name="delete" value="Delete">
+      </form>
+     <% }) %>
+
```

Comment:
Overall, really great job! I'm happy to see you're indenting and spacing is consistent too!